src/main/asciidoc/index.adoc

```diff
@@ -2,22 +2,30 @@
 
 The Service Resolver library is a plugin that lets Vert.x clients call services using logical service names instead of network addresses. The service resolver is also able to perform client side load balancing with the usual strategies.
 
-== Getting started with Vert.x HTTP client
+== Getting started with the Vert.x HTTP client
 
-Given a resolver, you can configure a Vert.x HTTP client to use it thansk to an HTTP client builder
+Given a resolver, you can configure a Vert.x HTTP client to use it thanks to an HTTP client builder.
 
 [source,java]
 ----
 {@link examples.ServiceResolverExamples#configuringHttpClient}
 ----
 
-A service is addressed with a `{@link io.vertx.serviceresolver.ServiceAddress}` instead of a `{@link io.vertx.core.net.SocketAddress}`
+A service is addressed with a `{@link io.vertx.serviceresolver.ServiceAddress}` instead of a `{@link io.vertx.core.net.SocketAddress}`.
 
 [source,java]
 ----
 {@link examples.ServiceResolverExamples#usingHttpClient}
 ----
 
+[TIP]
+====
+To integrate Vert.x Service Resolver with the Vert.x Web Client:
```

Comment:
I think we should get a section (even if it is copy/paste) "Getting started with the Vert.x Web client"

Reply:
The problem is we can't get an example without introducing a dependency to vertx-web-client

Reply:
we can work around with inline code without a class

Reply:
Done